rise.py

```diff
@@ -1 +1,2 @@
-print('rise and shine')
\ No newline at end of file
+print('rise and shine')
+print('Lets get started 
```

Comment:
start discussion here box....

Reply:
yes. good start. missing parenthesis,
```suggestion
print('Lets get started.') 
```